apps/whale-api/src/module.api/poolpair.fees.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import BigNumber from 'bignumber.js'
+import { EstimatedDexFees } from '@defichain/whale-api-client/dist/api/poolpairs'
+import { PoolPairInfo } from '@defichain/jellyfish-api-core/dist/category/poolpair'
+
+@Injectable()
+export class PoolPairFeesService {
+  constructor (
+    protected readonly rpcClient: JsonRpcClient
+  ) {
+  }
+
+  public async getDexFeesPct (poolPair: PoolPairInfo, poolPairId: string, fromToken: string, toToken: string): Promise<EstimatedDexFees | undefined> {
```

Comment:
hmm.. i think can move this to `poolswap.pathfinding.service.ts` as a private function??

Reply:
@eli-lim what do you think? I rmb we had a discussion that it's better to be in another service CC: @canonbrother

Reply:
Hmm I think we can put all dexFee-related fetching / calculations in here so all the logic is consolidated in one place. Expose them as helpers methods to Poolswap.pathfinding.service and beyond.